lib/user.rb

```diff
@@ -0,0 +1,8 @@
+class User
+    attr_reader :name, :learn, :jokes, :learn, :tell
```

Comment:
based on our interaction pattern, `learn` and `tell` would have been two methods to create yourself, because there was other work to be done within them